plugins/importexport/csv/CSVImportExportPlugin.inc.php

```diff
@@ -93,179 +102,328 @@ function executeCLI($scriptName, &$args) {
+							$emailAddress = $matches[4];
 						} else {
-							echo __('plugins.importexport.csv.noSeries', array('seriesPath' => $seriesPath)) . "\n";
+							$emailAddress = $contactEmail;
 						}
+					}
+					$author = $authorDao->newDataObject();
+					$author->setSubmissionId($submissionId);
+					$author->setUserGroupId($authorGroup->getId());
+					$author->setGivenName($givenName, $locale);
+					$author->setFamilyName($familyName, $locale);
+					$author->setEmail($emailAddress);
+					if ($firstAuthor) {
+						$author->setPrimaryContact(1);
+						$firstAuthor = false;
+					}
+					$author->setData('publicationId', $publicationId);
+					$authorDao->insertObject($author);
+				} // Authors done.
+
+				$publication->setData('abstract', $abstract, $locale);
+				$publication->setData('title', $title, $locale);
+				$publicationDao->updateObject($publication);
+
+				// Submission is done.  Create a publication format for it.
+				$publicationFormat = $publicationFormatDao->newDataObject();
+				$publicationFormat->setData("submissionId", $submissionId);
+				$publicationFormat->setData("publicationId", $publicationId);
+				$publicationFormat->setPhysicalFormat(false);
+				$publicationFormat->setIsApproved(true);
+				$publicationFormat->setIsAvailable(true);
+				$publicationFormat->setProductAvailabilityCode('20'); // ONIX code for Available.
+				$publicationFormat->setEntryKey('DA'); // ONIX code for Digital
+				$publicationFormat->setData('name', 'PDF', $submission->getLocale());
+				$publicationFormat->setSequence(REALLY_BIG_NUMBER);
+
+				if ($doi) {
+					$publicationFormat->setStoredPubId('doi', $doi);
+				}
+
+				$publicationFormatId = $publicationFormatDao->insertObject($publicationFormat);
+
+				// Create a publication format date for this publication format.
+				$publicationDate = $publicationDateDao->newDataObject();
+				$publicationDate->setDateFormat('05'); // List55, YYYY
+				$publicationDate->setRole('01'); // List163, Publication Date
+				$publicationDate->setDate($year);
+				$publicationDate->setPublicationFormatId($publicationFormatId);
+				$publicationDateDao->insertObject($publicationDate);
+
+				// Submission File.
+				import('lib.pkp.classes.file.TemporaryFileManager');
+				import('lib.pkp.classes.file.FileManager');
+				import('lib.pkp.classes.core.Core');
+
+				// Submission File.
+				$fileManager = new FileManager();
+				$extension = $fileManager->parseFileExtension($pdfUrl);
+				$dirNames = Application::getFileDirectories();
+				$submissionDir = sprintf(
+					'%s/%d/%s/%d',
+					str_replace('/', '', $dirNames['context']),
+					$press->getId(),
+					str_replace('/', '', $dirNames['submission']),
+					$submissionId
+				);
+
+				$temporaryFileManager = new TemporaryFileManager();
+				$temporaryFilename = $temporaryFileManager->getBasePath() . uniqid() . ".{$extension}";
+				$filePath = getcwd() . "/plugins/importexport/csv/submissionPdfs/" . $pdfUrl;
```

Comment:
It's pretty unusual for us to direct people to place import files directly within the OJS installation somewhere -- it's far likelier that they'll be working somewhere else. Rather than havin ghte PDFs placed inside the installation, you might instead specify that they should be in the same directory as the CSV file that's being imported.